source/blog/2017-09-27-webadmin-lldp.html.md

```diff
@@ -0,0 +1,30 @@
+---
+title: LLDP information in webadmin
+author: amusil
+tags: webadmin, networking
+date: 2017-09-27 09:00:00 CET
+---
+
+In the oVirt 4.2.0 we have introduced support for LLDP protocol. 
+LLDP stands for Link Layer Discovery Protocol, which is protocol used by network devices for advertising identity and capabilities to neighbors on a LAN. [More information about LLDP](https://learningnetwork.cisco.com/docs/DOC-26851).
+
+READMORE
+
+### Useful information
+
+The information gathered by protocol can be used for better network configuration. UI shows most important informations e.g. System name, Port description and VLAN IDs. Administrator can set VLAN ID according to switch configuration. In case of some problem Port description can become handy as it shows switch port number to which is network interface connected.
+
+
+### Where to find this information
+
+In the **Administration Portal**. From the vertical menu select **Compute** > **Hosts** and click on **Link (Name)** of your desired host. Then select **Network Interfaces** > **Setup Host Networks**. By hovering over interfaces appropriate tooltil will show with it's information. 
+
+## Non-UI approach
+
+The LLDP information is also exposed via REST API. More information is available in [REST API documentation](http://ovirt.github.io/ovirt-engine-api-model/4.2/#services/link_layer_discovery_protocol).
+
+
+### Some screenshots
+
+![screen](../images/blog/2017-09-27/1.png)
```

Comment:
The picture is good, but would a picture show an example all displayable LLDP information nicer? Can I assist in creating this picture?

Reply:
@dominikholler should the bond-related image have included a reference to the link-aggregation lldp frame? do you have an idea why it did not show up?

Reply:
According to the [source code](https://gerrit.ovirt.org/#/c/82560/18/frontend/webadmin/modules/webadmin/src/main/java/org/ovirt/engine/ui/webadmin/section/main/view/popup/host/panels/ItemInfoPopup.java@276) some LinkAggregation802 Information should be shown. I guess that this information is not included in the (real or faked?) output of lldp-tool.
If the output is faked, it should be easy to add this information.